app/src/main/java/com/android/sample/ui/composables/TopBar.kt

```diff
@@ -26,16 +31,21 @@ import com.android.sample.ui.theme.DarkBlue
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun TopBar(navigationActions: NavigationActions, @StringRes title: Int) {
-  TopAppBar(
-      title = {
-        Text(
-            text = stringResource(id = title),
-            fontSize = 20.sp,
-            fontWeight = FontWeight.Bold,
-            color = Color.White,
-            modifier = Modifier.testTag("ScreenTitle"))
-      },
-      navigationIcon = { ArrowBack(navigationActions, Color.White) },
-      colors = TopAppBarDefaults.smallTopAppBarColors(containerColor = DarkBlue),
-      modifier = Modifier.testTag("TopBar"))
+  val gradientBrush = Brush.horizontalGradient(colors = listOf(TitleBlue, TopBarBlue))
```

Comment:
Maybe you could put this in Color.kt, to be able to reuse it for others screens

Reply:
Done